src/lavatory/utils/artifactory.py

```diff
@@ -76,18 +85,19 @@ def purge(self, dry_run, artifacts):
 
         for artifact in artifacts:
             artifact_path = "{}/{}".format(artifact['path'], artifact['name'])
-            LOG.info("  {} purge {}:{}".format(mode, self.repo_name, artifact_path))
+            LOG.info("  %s purge %s:%s", mode, self.repo_name, artifact_path)
             if dry_run:
                 purged += 1
             else:
                 try:
                     self.artifactory.delete(artifact_path)
                     purged += 1
-                except Exception as error:
+                except requests.exceptions.BaseHTTPError as error:
```

Comment:
Is this the right exception? Maybe there is a better `party` exception we can catch or something?

Reply:
Party uses .raise_for_status() which can raise any one of the exceptions. I think I could group most common ones.